packages/plugin-core/src/_extension.ts

```diff
@@ -735,85 +733,14 @@ export async function _activate(
       }
     }
 
-    const backupPaths: string[] = [];
-    let keybindingPath: string;
-
-    // check vim keybindings ^j7zygfkbxjnq
```

Comment:
any docs we need to clean up with this block anchor removal?

Reply:
https://github.com/dendronhq/dendron-docs/pull/22